### PR TITLE
#4671 [Manager] Paper element's metrics should always being displayed when the metrics switch is on

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4671)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Pipeline/PipelinePropertyView/MetricsPanel.js [hunk 1]
```diff
@@ -51,13 +51,11 @@ const MetricsPanel = props => {
...
         <ExpansionPanelDetails>
-          {Object.keys(metrics).map((metricsKey, index) => {
+          {Object.keys(metrics).map(key => {
             return (
-              <>
-                <Typography variant="h6" key={index}>
-                  {metricsKey}
-                </Typography>
-                {metrics[metricsKey].meters.map((meter, index) => {
+              <React.Fragment key={key}>
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/PipelinePropertyView/MetricsPanel.js:56]: React key warning...

File: ohara-manager/client/src/components/Pipeline/PipelinePropertyView/PipelinePropertyView.js [hunk 2]
```diff
@@ -106,7 +106,9 @@ const PipelinePropertyView = props => {
         setTags={setTags}
         setFullTagViewDialogOpen={setIsOpen}
       />
-      <NodePanel tasksStatus={settings?.tasksStatus} />
+      {element.kind !== KIND.topic && (
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/PipelinePropertyView/PipelinePropertyView.js:109]: Topic doesn't have taskStatus... but it has "partitionInfos". @eechih but this is not planned in our current UI yet. I will open an issue to track this and for now, topic won't display any "Node" related infomation

Comment (eechih) [reply ohara-manager/client/src/components/Pipeline/PipelinePropertyView/PipelinePropertyView.js:109]: OK, thanks :)

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/PipelinePropertyView/PipelinePropertyView.js:109]: #4677 I will leave a comment as well

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manger-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**